services/web/server/src/simcore_service_webserver/director_v2/_handlers.py

```diff
@@ -69,83 +70,100 @@ class _ComputationStarted(BaseModel):
 @permission_required("services.pipeline.*")
 @permission_required("project.read")
 async def start_computation(request: web.Request) -> web.Response:
-    req_ctx = RequestContext.parse_obj(request)
-    computations = ComputationsApi(request.app)
+    # pylint: disable=too-many-statements
```

Comment:
I agree with pylint here... the maintainability of this method is not good.

Reply:
Yes, I have some ideas on how to refactor, will do, when the right time comes.